src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.cs

```diff
@@ -143,7 +143,10 @@ public override void VisitProperty(IPropertySymbol symbol)
 
                 AddCustomModifiersIfRequired(symbol.RefCustomModifiers);
 
-                symbol.Type.Accept(this.NotFirstVisitor);
+                var symbolType = ShouldBeNullableAnnotated(symbol, FlowAnalysisAnnotations.MaybeNull)
```

Comment:
>ShouldBeNullableAnnotated(symbol, FlowAnalysisAnnotations.MaybeNull) [](start = 33, length = 68)

It doesn't feel right to call this when ```symbol.Type``` is a value type.

Reply:
Fixed in 57b130db